apps/dashboard/test/apps/nav_bar_test.rb

```diff
@@ -88,53 +88,93 @@ def setup
     assert_equal 4,  result[0].apps.size
   end
 
-  test "NavBar.items should return navigation group with sort=true when nav_item has apps and title property" do
+  test "NavBar.items should ignore nav_item when nav_item has apps token that matches multiple apps without a title property" do
+    nav_item = {
+      apps: "sys/*"
+    }
+
+    result = NavBar.items([nav_item])
+    assert_equal [],  result
+  end
```

Comment:
It's not clear to me why we'd want this behavior. It'll take me a second to replicate what the current behavior is (if it differs at all)

Reply:
This is not new. I am just adding a test for it.
The reason that is ignored is because the system does not have a title to create a menu for all the apps that match.

Reply:
Got it, thanks.